src/figuresClasses.ts

```diff
@@ -1,19 +1,96 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
Create a union type for shape and color because you have a limited list of possible properties